### PR TITLE
Add Flagstat and FASTQC emails to the EDSL and pipeline

### LINKED ISSUE #11: Email notifications option for QC

### PR DESCRIPTION
**Pull request by ihodes**
Fixes #11 

WIP: adding fastqc emails next 

Thanks to @arman for the lovely summarizing shell script used herein; happy to have emails back in the pipeline!

### CODE REVIEW

**Comment by ihodes** [comment]
@smondet this is ready for your eyes now! I still need to test this and make sure the emails send, though.

File: src/lib/command_line.ml [hunk 1]
```diff
@@ -145,7 +148,6 @@ let pipeline_term
       in
       pure begin fun
         (`Dry_run dry_run)
-        (`Mouse_run mouse_run)
```
Comment (smondet) [inline src/lib/command_line.ml:148]: Why did the mouse run option disapear?

**Comment by ihodes** [comment]
After a lot of bash debugging and submitting jobs, this appears to be working very nicely! @smondet ready when you are

**Comment by armish** [comment]
FastQC e-mailing code looks much cleaner now! Thanks for all the debugging/cleaning, @ihodes!

File: src/lib/command_line.ml (hunk 1 above)
Comment (smondet) [reply src/lib/command_line.ml:148]: Oh the mouse came back, this was an outdated diff.

**Pull request merged**